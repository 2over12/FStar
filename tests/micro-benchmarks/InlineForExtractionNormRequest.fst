module InlineForExtractionNormRequest

inline_for_extraction
let id (x:'a) = x

let test (x:int) = norm [primops] (id 0 + 1)
