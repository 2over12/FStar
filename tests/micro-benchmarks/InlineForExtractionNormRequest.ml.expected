open Prims
let id : 'a . 'a -> 'a = fun x -> x
let (test : Prims.int -> Prims.int) = fun x -> Prims.int_one